### PR TITLE
Adding updates to lre07/v2 

### PR DESCRIPTION
**Pull request by s-mousmita**
Correcting bug in lre07/v2 and adding GPU support to nnet posterior computation part of I-vector scripts.

### CODE REVIEW

**Comment by david-ryan-snyder** [comment]
@s-mousmita, thanks! Could you let me know if it's ready to review?

**Comment by s-mousmita** [comment]
@david-ryan-snyder  Yes, please.

File: egs/lre07/v1/lid/extract_ivectors_dnn.sh [hunk 1]
```diff
@@ -60,24 +82,28 @@ sdata_dnn=$data_dnn/split$nj;
 utils/split_data.sh $data_dnn $nj || exit 1;
           
 
-# Set up features.
+# Set up language recognition features
 feats="ark,s,cs:apply-cmvn-sliding --norm-vars=false --center=true --cmn-window=300 scp:$sdata/JOB/feats.scp ark:- | add-deltas-sdc ark:- ark:- | select-voiced-frames ark:- scp,s,cs:$sdata/JOB/vad.scp ark:- |"
-
+# set uo nnet features
```
Comment (david-ryan-snyder) [inline egs/lre07/v1/lid/extract_ivectors_dnn.sh]: Could you fix the typo? uo -> up

**Comment by s-mousmita** [comment]
@david-ryan-snyder I have addressed the comments. Please check.

**Comment by david-ryan-snyder** [comment]
Looks good. Just two small things and we can merge it:

Could you remove trailing whitespace from your code? An easy way to make sure you never have trailing whitespace in your code is to add a line like this to your .vimrc:

`autocmd BufWritePre *.cc,*.h,*.sh,*.pl,*.py :%s/\s\+$//e`

Once you get rid of the trailing whitespace, could you rebase this so it's all in one commit? Also, can you rewrite the commit message so that it looks more like the recent commit messages in Kaldi? E.g., something like `[scripts, egs] In egs/lre07/v2 fixing mistake in training list in run.sh and adding GPU support for language ID iVector scripts that use a DNN"`

**Comment by s-mousmita** [comment]
@david-ryan-snyder Thanks for the review. I have removed the trailing whitespace and updated the commit message.

**Comment by david-ryan-snyder** [comment]
@s-mousmita, looks good to me.

@danpovey, I think we can merge this now.

**Comment by danpovey** [comment]
Thanks, merging.

**Comment by s-mousmita** [comment]
@david-ryan-snyder, @danpovey, Thanks.

**Pull request merged**